_data/locales/tra-ch.yml

```diff
@@ -0,0 +1,31 @@
+tra-ch:
```

Comment:
I think this should be `zh-hant` (or `zh-tw` if you don't mind more common and jurisdiction-denoting language code--the existing simplified Chinese translation uses `zh-cn`). I've never seen `tra-ch` or similar before.

Reply:
Alright. Thank you for your guiding.